source/installation/kafka-deployments/install-kafka-cluster.txt

```diff
@@ -37,13 +37,13 @@ Steps
       a. Download the source connector archive for your source database:
 
          - `MySQL Connector plugin archive 
-           <https://repo1.maven.org/maven2/io/debezium/debezium-connector-mysql/2.4.1.Final/debezium-connector-mysql-2.4.1.Final-plugin.tar.gz>`__
+           <https://repo1.maven.org/maven2/io/debezium/debezium-connector-mysql/2.7.0.Final/debezium-connector-mysql-2.7.0.Final-plugin.tar.gz>`__
```

Comment:
[Suggestion]: Given that all these connectors match to the same version, would it make sense to add it as a variable in `snooty.toml`?

Reply:
Great suggestion!